test/integration/undo-redo/index.spec.ts

```diff
@@ -42,42 +38,3 @@ test('test basic undo and redo', async ({ page, browserName, api }) => {
   // Redo should bring back text field
   await expect(canvasInputLocator).toHaveCount(3);
 });
-
-test('test batching quick actions into single undo entry', async ({ page, browserName, api }) => {
```

Comment:
is it ok to delete this test? we can add more comprehensive testing of undo/redo - perhaps one for changing views and one for the upcoming text input logic?